utils/make-patch-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+url=$1
+echo "URL: $url"
+sha_sum=$(cd bin; sha256sum calico-felix)
+
+echo
+echo "Run the following script on the target server to patch the"
+echo "/opt/calico-felix/calico-felix binary:"
+echo
+echo "curl -o calico-felix $url &&"
+echo "    if echo '$sha_sum' | sha256sum --status --check; then \\"
```

Comment:
Should the `sha256sum` invocation include `calico-felix` somewhere?  Otherwise how does it know what file to check?